client/src/components/CardInfos.js

```diff
@@ -1,43 +1,44 @@
 import React from 'react';
-import styled from 'styled-components/macro';
+import styled, { css } from 'styled-components';
 import ReactDom from 'react-dom';
 import PropTypes from 'prop-types';
 import AddButton from '../components/AddButton';
 import AudienceScore from '../assets/Audience_score';
+import backdrop_poster from '../images/backdrop_poster.png';
 
-export default function CardInfos({
-  open,
-  onClose,
-  movieText,
-  movieName,
-  release,
-  background,
-  baseUrl,
-  score,
-  addToWatchList,
-}) {
+export default function CardInfos({ open, onClose, addToWatchList, movie }) {
   if (!open) return null;
+  const baseUrl = 'https://image.tmdb.org/t/p/original/';
+  const release = movie.first_air_date || movie.release_date;
+  let check;
+  movie.poster_path === null && movie.backdrop_path === null
+    ? (check = true)
+    : (check = false);
```

Comment:
This was hard for me to understand. 
Probably the following will be sufficient as the check on the right will either return true or false thus your check will hold one of each.  
`const check = movie.poster_path === null && movie.backdrop_path === null`

You could also reconsider the name - what does it check for?

Reply:
Funny, Thomas made the very same suggestions just a couple of hours ago :) The line provides a boolean so I can handle the background/poster with own pictures whenever the API fails to provide data. Good point about changing the name :)